kore/src/Kore/Internal/SideCondition/SideCondition.hs

```diff
@@ -3,6 +3,8 @@ Copyright   : (c) Runtime Verification, 2020
 License     : NCSA
 -}
 
+{-# LANGUAGE StrictData #-}
```

Comment:
Why isn't this just `Strict`? Also, the only data type in this module was made lazy, so why not just `NoStrict` entirely?

Reply:
From the link you gave me, I assumed we'd also like to make all modules `StrictData` by default, in the future. Does adding `NoStrict` bypass `StrictData` as well?